app/controllers/game.js

```diff
@@ -8,17 +8,32 @@ const Game = mongoose.model('Game');
  * @param {string} res - response send to client end.
  */
 
-exports.start = (req, res) => {
+exports.save = (req, res) => {
   const newGame = new Game();
   // game_id should use _id
   // this is just a sample data, should be updated for real user
-  newGame.gameOwnerId = req.body.gameOwnerId;
+  newGame.gameOwnerEmail = req.body.gameOwnerEmail;
   newGame.players = req.body.players;
   newGame.gameWinner = req.body.gameWinner;
   newGame.date = new Date();
+  newGame.save((err, success) => {
+    if (err) {
+      throw (err);
```

Comment:
do not throw error here, this will hang the request
`res.status(500).send({ error: 'An error occured' });`